### PR TITLE
LNP-1063: header improvements + add translations links

### PR DESCRIPTION
**Pull request by madebyzak**
This PR;
- Updates the header with the translations links (currently behind feature flag)
- Provides a header that will, in future, provide a dropdown i.e. when the Settings page related links are ready to be integrated into the header.
- Cleans up the class names and styles. 

https://dsdmoj.atlassian.net/browse/LNP-1063

### CODE REVIEW

File: server/views/partials/header.njk [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
-          <li class="hmpps-header__navigation__item" id="user-name">
-              <span>{{ user.idToken.name }}</span>
-          </li>
-        </ul>
-      </nav>
-    {% endif %}
+    {{ govukNav(user, userSvg, isTranslationsEnabled, currentLng) }}
   </div>
-</header>
+</header>
```
Comment (psoleckimoj) [inline server/views/partials/header.njk:31]: Missing trailing newline

File: server/views/components/nav/macro.njk [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        </div>
+      </div>
+    </nav>
+  {% endif %}
+{% endmacro %}
+
+{% macro translationsNav(isTranslationsEnabled, currentLng) %}
+  {% if isTranslationsEnabled %}
+    <div class="launchpad-home-header__navigation__item launchpad-home-header__navigation__item__translations">
+      {% set translations = [
```
Comment (psoleckimoj) [inline server/views/components/nav/macro.njk:21]: Could possibly move this list into the route or somewhere and drag through from config 🤔 Saves config being inside a nunjucks template?

Comment (madebyzak) [reply server/views/components/nav/macro.njk]: Good idea 😄

**Review by psoleckimoj — APPROVED** [review]
LGTM!

**Pull request merged**